index.php

```diff
@@ -7,30 +7,45 @@
 <main id="main-content">
 
   <!-- main posts loop -->
-  <section id="posts">
+  <section id="posts" class="container">
+    <div class="row">
 
 <?php
 if( have_posts() ) {
   while( have_posts() ) {
     the_post();
-?>
 
-    <article <?php post_class(); ?> id="post-<?php the_ID(); ?>">
+    $single_row = get_post_meta($post->ID, '_igv_single_row', true);
+    $top_margin = get_post_meta($post->ID, '_igv_top_margin', true);
+    $percent_width = get_post_meta($post->ID, '_igv_percent_width', true);
+    $percent_rotate = get_post_meta($post->ID, '_igv_percent_rotate', true);
+
+    $single_row = empty($single_row) ? '' : $single_row;
+    $top_margin = empty($top_margin) ? '0' : $top_margin;
+    $percent_width = empty($percent_width) ? '100' : $percent_width;
+    $percent_rotate = empty($percent_rotate) ? '0' : $percent_rotate;
+?>
 
-      <a href="<?php the_permalink() ?>"><?php the_title(); ?></a>
+      <article <?php 
+        if ($single_row == 'on') {
+          post_class('text-align-center col col-s-12'); 
+        } else {
+          post_class('text-align-center col col-s-12 col-m-6');
+        }
+      ?> id="post-<?php the_ID(); ?>" style="margin-top: <?php echo $top_margin; ?>px">
 
-      <?php the_content(); ?>
+        <a href="<?php the_permalink() ?>">
+          <?php the_post_thumbnail('gallery', array('style'=>'max-width: ' . $percent_width . '%; transform: rotate(' . $percent_rotate . 'deg)')); ?>
```

Comment:
doing a simple `rotate()` breaks out of bounding box so can overlap text or other content nearby. Perhaps needs to be rotated with javascript: https://stackoverflow.com/questions/19794171/css-rotate-element-while-staying-inside-container. In a scaffold I'd imagine that requires saving data values and a class to `.each` over